rethinkdb/datadog_checks/rethinkdb/check.py

```diff
@@ -71,45 +101,14 @@ def connect_submitting_service_checks(self):
             raise
         else:
             self.service_check(self.SERVICE_CHECK_CONNECT, self.OK, tags=tags)
+        finally:
+            self._conn = None
```

Comment:
in future, wdyta caching the connection?

Reply:
Yes, we could, though A/ it'd need to handle connection losses, B/ opening a connection once every 15s sounds like an okay burden to put on RethinkDB, and C/ I don't think there's a particular benefit in reusing a RethinkDB connection (unlike for HTTP requests which gets us connection pooling / keep-alive, for example).

So I sticked to simplicity for now, but we could consider it.